common/changes/@rushstack/ts-command-line/user-danade-PreventAmbiguousAbbreviations_2023-10-23-20-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/ts-command-line",
+      "comment": "Consider parent tool and action parameters when determining ambiguous abbreviations",
```

Comment:
Can you give an example in the changelog? This isn't really clear without context.